.github/workflows/smoke-test-daily.yml

```diff
@@ -44,36 +46,24 @@ jobs:
 
             - name: Run 'Update WooCommerce' test.
               working-directory: plugins/woocommerce
-              id: e2e-update
               env:
                   UPDATE_WC: true
-              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js update-woocommerce.spec.js
+              run: pnpm exec playwright test --config=tests/e2e-pw/daily.playwright.config.js update-woocommerce.spec.js
 
             - name: Run the rest of E2E tests.
               timeout-minutes: 60
               working-directory: plugins/woocommerce
-              id: e2e
               env:
-                  E2E_MAX_FAILURES: 15
-              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js basic.spec.js
+                  E2E_MAX_FAILURES: 25
```

Comment:
I upped the max failures to 25 (PR tests only have 15 as their max failures) to take into account some flakiness on the daily smoke test site.